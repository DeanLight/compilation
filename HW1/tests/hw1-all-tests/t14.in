%streams
stream

endstream
stream
7¬÷gYוֺ¼¶tg מ9®{ֿ¥PhYˆhת]Dַ.”h]‰ּ
sr3—³;V¶ w‹!-ˆ`]ײ›DI…¥ר״ס c#¿»¢רb/=¦
endstream stream
endstream
endstream
stream
adkfjhakdfjh