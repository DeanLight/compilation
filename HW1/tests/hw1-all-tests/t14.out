1 COMMENT %streams
4 STREAM 
8 STREAM 7¬÷gYוֺ¼¶tg מ9®{ֿ¥PhYˆhת]Dַ.”h]‰ּ
sr3—³;V¶ w‹!-ˆ`]ײ›DI…¥ר״ס c#¿»¢רb/=¦
10 STREAM endstream
Error unclosed stream
